src/app/components/activity/feed/devlog-post/text/text.ts

```diff
@@ -59,9 +58,13 @@ export default class AppActivityFeedDevlogPostText extends Vue {
 		}
 	}
 
-	expand() {
+	async expand() {
 		this.feed.setItemOpen(this.item, true);
```

Comment:
Don't we want to the set feed item as open only after the article was loaded in?

Reply:
Nah, we need to open it up to show the loading graphic. If it's not set to "open" it won't show that it's loading. I guess we can change it so that it'll open up if it's loading OR if it's open? It would just complicate the checks a bit more, so I kept it simple like this.

I suppose there could be a race condition that they try opening the article and then move away from the page and the article never gets content somehow...